kafka-ui-react-app/src/components/Brokers/BrokersList/__test__/BrokersList.spec.tsx

```diff
@@ -134,12 +134,20 @@ describe('BrokersList Component', () => {
         }));
       });
 
-      it('renders empty table', async () => {
```

Comment:
why are we deleting the empty table case ?

Reply:
fixed